go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/reugn/go-streams
 
 go 1.15
+
+require github.com/nats-io/stan.go v0.7.0 // indirect
```

Comment:
NATS deps should go under it's own go.mod file.

Reply:
Yes absolutely. Not sure how these got into the root of the project.